scripts/check_secrets.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+# Copyright 2020 Google
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Check if secrets are available for multiple CI's
+
+set -x
+echo $GITHUB_BASE_REF
```

Comment:
Please either remove this debug logging or make it easier to understand by labeling the output, e.g.

```bash
echo "GITHUB_BASE_REF: ${GITHUB_BASE_REF}"
```